### ISSUE TITLE
Fix doxygen support to include member groups

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1237]
Note that the spec says three slashes, but their example  has only two slashes.

Once this is done, we can try turning on sp_cmt_cpp_start in Uncrustify.Common-CStyle.cfg.

I want to turn on sp_cmt_cpp_start=add and sp_cmt_cpp_doxygen=true and have it not break our existing non-spec doxygen comments that only use //@{ and not ///@{.

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10056-UNI-1346.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10056-UNI-1346.cpp"
index 1400adf..524dbce 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10056-UNI-1346.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10056-UNI-1346.cpp"
@@ -1,24 +1,24 @@
 // Fix doxygen support to include member groups
 
 // See http://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup
 
 // Note that the spec says three slashes, but their example has only two slashes.
 
 // Once this is done, we can try turning on sp_cmt_cpp_start in Uncrustify.Common-CStyle.cfg.
 
 /// Bucket allocator is used for allocations up to 64 bytes of memory.
 /// It is represented by 4 blocks of a fixed-size "buckets" (for allocations of 16/32/48/64 bytes of memory).
 /// Allocation is lockless, blocks are only growable.
 class Class
 {
 public:
     ///@{ Doxygen group 1
     virtual void*  Foo();
     virtual void*  Bar();
     ///@}
 
-    //@{ Doxygen group 2
+    // @{ Doxygen group 2
     virtual void*  Foo();
     virtual void*  Bar();
-    //@}
+    // @}
 }
```

Debug support files:
---
[10056-UNI-1346.cpp](https://gist.githubusercontent.com/mihaipopescu/6fc1f469d58faa5764008f104226b2b0/raw/1400adff0e13c3fcbad5b95951a6e3dbdee6b24e/10056-UNI-1346.cpp)
[10056-UNI-1346.cpp.log](https://gist.githubusercontent.com/mihaipopescu/6fc1f469d58faa5764008f104226b2b0/raw/eae51e59829f65d0f92ccd96b112a1df84f60075/10056-UNI-1346.cpp.log)
[10056-UNI-1346.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/6fc1f469d58faa5764008f104226b2b0/raw/10a7a4b36703a90c669413807d892f9bf9669cfc/10056-UNI-1346.cpp.unc)
[UNI-1346.cfg](https://gist.githubusercontent.com/mihaipopescu/6fc1f469d58faa5764008f104226b2b0/raw/2157f08e0cdedd530c435afe980f80229a0d4b85/UNI-1346.cfg)
[UNI-1346.cpp](https://gist.githubusercontent.com/mihaipopescu/6fc1f469d58faa5764008f104226b2b0/raw/d054f623a3454d751c25ceab0e5a79b332114c02/UNI-1346.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by guy-maurel** [comment]
The current git version works pretty fine.
  https://github.com/uncrustify/uncrustify
Please test it and report!

**Comment by gmaurel** [comment]
Fixed at ed339ca86e656e478ea20f8547fc8440866103d5, Jan 14 18:53:26 2018
